src/Mvc/GetDocumentInsider/src/Commands/GetDocumentCommandWorker.cs

```diff
@@ -22,200 +46,325 @@ public static int Process(GetDocumentCommandContext context)
                 return 2;
             }
 
-            var services = GetServices(entryPointType, context.AssemblyPath, context.AssemblyName);
-            if (services == null)
+            static string FormatException(Exception exception) => $"{exception.GetType().FullName}: {exception.Message}";
```

Comment:
consider putting this after the return statement to separate it a bit. It's a little jarring to see a single-line local function defined in the middle of a method.

Reply:
I would really really strongly encourage the use of `Exception.ToString()` instead of hiding the information behind a verbose logging switch. 

By doing this you can optimize for users getting an experience that they troubleshoot themselves (failure in their own app), or can use to immediately report a problem to us (copy-paste the call stack).